### DISCUSSION TITLE
Hiding redundant query log?

### DISCUSSION THREAD

**Discussion by r1bnc** [AdguardTeam/AdGuardHome#3561] [Q&A] [answered]
I have this query, it spams the entire logs. Any possible way to hide this?
Also, anyone know what is this query?

![image](https://user-images.githubusercontent.com/8337343/132304286-105fdd9f-5c89-4374-b12c-cdc42a8749b2.png)

Thank you.

**Comment by ainar-g** [comment, marked as answer]
There is currently no way to filter the query log by query type, unfortunately.  This feature is planned for one of the future releases.  See #594.

**Reply by r1bnc** [reply]
Thank you for the reply, now I need to find out which host is generating those endless NS req.
